[2014-08-13 20:33:11] log.ERROR: 500 - View [site.contacts.edit] not found. @ /contacts/1/edit
exception 'InvalidArgumentException' with message 'View [site.contacts.edit] not found.' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:135
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php(76): Illuminate\View\FileViewFinder->findInPaths('site.contacts.e...', Array)
#1 /home/htdocs/turnos1/vendor/robclancy/presenter/src/Robbo/Presenter/View/Environment.php(46): Illuminate\View\FileViewFinder->find('site.contacts.e...')
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Robbo\Presenter\View\Environment->make('site.contacts.e...')
#3 /home/htdocs/turnos1/app/controllers/ContactsController.php(118): Illuminate\Support\Facades\Facade::__callStatic('make', Array)
#4 /home/htdocs/turnos1/app/controllers/ContactsController.php(118): Illuminate\Support\Facades\View::make('site.contacts.e...')
#5 [internal function]: ContactsController->edit('1')
#6 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#7 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('edit', Array)
#8 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'edit', Array)
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1')
#10 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#11 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#12 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#16 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#17 {main} [] []
[2014-08-13 21:24:10] log.ERROR: 500 - include(/home/htdocs/turnos1/app/controllers/EquipsController.php): failed to open stream: No such file or directory @ /admin/equips
exception 'ErrorException' with message 'include(/home/htdocs/turnos1/app/controllers/EquipsController.php): failed to open stream: No such file or directory' in /home/htdocs/turnos1/vendor/composer/ClassLoader.php:382
Stack trace:
#0 /home/htdocs/turnos1/vendor/composer/ClassLoader.php(382): Illuminate\Exception\Handler->handleError(2, 'include(/home/h...', '/home/htdocs/tu...', 382, Array)
#1 /home/htdocs/turnos1/vendor/composer/ClassLoader.php(382): Composer\Autoload\includeFile()
#2 /home/htdocs/turnos1/vendor/composer/ClassLoader.php(274): Composer\Autoload\includeFile('/home/htdocs/tu...')
#3 [internal function]: Composer\Autoload\ClassLoader->loadClass('EquipsControlle...')
#4 [internal function]: spl_autoload_call('EquipsControlle...')
#5 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Container/Container.php(308): ReflectionClass->__construct('EquipsControlle...')
#6 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Container/Container.php(250): Illuminate\Container\Container->build('EquipsControlle...', Array)
#7 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(478): Illuminate\Container\Container->make('EquipsControlle...', Array)
#8 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(983): Illuminate\Foundation\Application->make('EquipsControlle...')
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#10 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#11 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#12 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#16 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#17 {main} [] []
[2014-08-13 21:24:32] log.ERROR: 404 - Exception @ /assets/css/images/controls.png
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-13 21:24:32] log.ERROR: 404 - Exception @ /assets/css/images/loading.gif
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
